### ISSUE TITLE
linear haskell `->.` operator

### ISSUE THREAD

**Issue by arknjl** [tonsky/FiraCode#918]
in linear haskell, `->.` is used as the linear version of the standard arrow `->`. this bugs me + others as it doesn’t look anything like the inherent mathematical symbol (U+22B8 MULTIMAP `⊸`).
visually speaking, it’s a stretch, so if it’s too jarring perhaps keep the tip the arrow for a `->o`-style thing with the tip de-emphasized

while being an extremely small use case, it would make the lives of three people out there slightly more beautiful

**Comment by tonsky** [comment]
Ok, I’ll look into it

**Comment by chroma04** [comment]
it’s been revealed that in upcoming GHC 12, linear types are finally officially coming to haskell!
although, the operator is now `#->`
could you do something with that?

**Comment by tonsky** [comment]
I’m not sure... `#->` looks pretty decent right now

**Comment by hellwolf** [comment]
Yes please provide for "⊸" character, it looks quite out of place at the moment because it falls back to a different font (Noto Sans Mono in this case):

![image](https://github.com/tonsky/FiraCode/assets/186660/3011daf3-b0c3-4d5b-a9ad-ebb201e6da29)

**Comment by jake-87** [comment]
Pinging this. Please add.

**Comment by ploddi** [comment]
Scala [Libretto](https://github.com/TomasMikula/libretto) also contains similar [linear functions](https://github.com/TomasMikula/libretto#how-to-type-the--symbol-used-in---and-). Ligatures will be welcome.
